src/enlyze/models.py

```diff
@@ -8,8 +8,12 @@
 import pandas
 
 
+class Model:
+    """Base Class for all models exposed via the :class:`~enlyze.client.EnlyzeClient`"""
+
+
```

Comment:
That being said, [a10bcc6](https://github.com/enlyze/enlyze-python/pull/16/commits/a10bcc6310ef0a82f22039c01e5997d0d1befb8a) removes it since its unnecessary